addons/dark-www/experimental_forums.css

```diff
@@ -92,3 +92,9 @@
   border-color: #606060 !important;
   background: transparent;
 }
+
+.uploadStatus {
+  height: 19px;
+  color: black;
+  padding: 1.5px;
```

Comment:
I don't think this is the right way to do it, why are you specifying the height and padding in dark mode css?

Reply:
because you don't notice it unless you are in dark mode. it only has a background in dark mode

Reply:
A better fix would be removing the background.

Reply:
@mxmou done